test/test_view.c

```diff
@@ -54,23 +54,39 @@ int main(int argc, char *argv[])
 
     while ((c = getopt(argc, argv, "IbDCSl:t:i:o:N:BZ:@:")) >= 0) {
         switch (c) {
-        case 'S': flag |= 1; break;
+        case 'I': ignore_sam_err = 1; break;
```

Comment:
Adding usage is most welcomed, thanks, but as for prior comment, why have the options changed order?  It doesn't look to be for consistency with the usage statement.